### PR TITLE
WIP: Test Docker multi-arch image

### PR DESCRIPTION
**Pull request by miekg**
Test PR to feel the waters; untested; this add a Makefile and docker file that builds a multi arch docker image - I'll be using this to test metallb at home.

### CODE REVIEW

**Comment by miekg** [comment]
fyi: just jotting down some docs for me

For my testing to works I also need to give the controller same treatment and then change the yaml to use my docker registry and (I think) add `hostNetwork = true`

File: arp-speaker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM alpine:latest
```
Comment (danderson) [inline arp-speaker/Dockerfile]: Check out the contents of the `dockerfiles/` directory, it has some templates for dev+prod containers for the metallb binaries. At some point you might want to try using those templates, which might work as well?

File: arp-speaker/Makefile [hunk 2]
```diff
@@ -0,0 +1,61 @@
+SYSTEM:=
```
Comment (danderson) [inline arp-speaker/Makefile]: I have to admit I'm really not a fan of the makefile format. I know it's super portable and available anywhere, but ugh it's such write-only code :(. That's why I used Fabric in the end, python is at least semi-maintainable.

But as I said in #47, if we can get all the stuff currently in fabfile into a makefile... I guess we could maybe switch to makefile. I'll hate it, but if it makes metallb easier to hack on...

**Comment by miekg** [comment]
Note: I have no expectation that any of this will be merged. Mostly me doing my thing whilst debugging arp on arm and having this mostly as an fyi.

**Comment by danderson** [comment]
This is a great FYI, and thank you for posting it! I'll translate it back into fabfile once I have a free moment, but this is a great guide for what the fabric command needs to do.

**Comment by miekg** [comment]
Code changes have been merged seperately. The Makefile and Dockerfiles are now in #59.

Closing this here.

**Pull request closed without merging**